windows/CommonSettings.props.example

```diff
@@ -11,6 +11,10 @@
          set to the root of your Python installation. If your Python installation
          does not contain debug libraries, debug build will not work. -->
         <PythonSupport>false</PythonSupport>
+		<!-- NOTE: If Matlab support is enabled, MatlabDir (below) needs to be
+         set to the root of your Matlab installation, and Python support has to
```

Comment:
Well, initially I tried to do it in a way that didn't matter if pycaffe support was enabled or not, but then while managing the dependencies and the linking process I found a bit strange having to add Python dependencies to this compilation. I think that it's better to remove them.